Assets/Game/Scripts/PlayerController.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using UnityEngine.InputSystem;
+
+public class PlayerController : MonoBehaviour
+{
+    [SerializeField] private Rigidbody2D _rb;
+    [SerializeField] private WheelJoint2D _rearWheel;
+    [SerializeField] private float _rotationSpeed = 1;
+    [SerializeField] private float _motorSpeed = 1000;
+    [SerializeField] private float _maxMotorTorque = float.PositiveInfinity;
+
+    private float _leanValue = 0;
+    private float _leanBackValue = 5f;
+    private float _leanForwardValue = -5f;
+
+    private void Start()
+    {
+        if (SystemInfo.deviceType == DeviceType.Handheld)
+        {
+            _leanBackValue = 10f;
+            _leanForwardValue = -10f;
```

Comment:
Also magic numbers. U can add some sort of [SerializeField] handledDeviceLeanModificator = 2f . End here, just multiply _leanBackValue and _leanForwardValue